orchagent/pfcactionhandler.cpp

```diff
@@ -436,13 +436,20 @@ PfcWdLossyHandler::PfcWdLossyHandler(sai_object_id_t port, sai_object_id_t queue
 {
     SWSS_LOG_ENTER();
 
-    string platform(getenv("platform"));
-
-    if (platform == CISCO_8000_PLATFORM_SUBSTRING)
+    const char *env_plat = getenv("platform");
+    if (env_plat == nullptr) 
```

Comment:
string platform = getenv("platform") ? getenv("platform") : ""  would be much cleaner